### PR TITLE
[JBIDE-23862] using latest osjc so that paths to "oc" may have " "

### PR DESCRIPTION
**Pull request by adietish**
# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by adietish** [comment]
@jeffmaury @bdshadow please +1 this which includes the latest openshift-restclient-java in JBT. 
Given that this is a -SNAPSHOT release I have to turn the enforcer rule off and make sure we release and use a final version of the client before the freeze.

**Comment by adietish** [comment]
testPR

**Comment by bdshadow** [comment]
testPR

**Comment by adietish** [comment]
testPR

**Comment by adietish** [comment]
@bdshadow build finally worked this time. Could you please review this?

File: plugins/org.jboss.tools.openshift.client/.classpath [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <classpath>
-	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-5.3.0.Final.jar" sourcepath="lib/openshift-restclient-java-5.3.0.Final-sources.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry exported="true" kind="lib" path="lib/openshift-restclient-java-5.5.0-SNAPSHOT.jar" sourcepath="/openshift-restclient-java"/>
```
Comment (bdshadow) [inline plugins/org.jboss.tools.openshift.client/.classpath]: @adietish why has the `sourcepath` changed? Does it matter?

Comment (adietish) [inline plugins/org.jboss.tools.openshift.client/.classpath]: hmm, not really, but good catch, i'll set it back to the source-jar

**Comment by adietish** [comment]
@bdshadow corrected it

**Pull request merged**